src/Microsoft.Framework.Caching.Abstractions/project.json

```diff
@@ -1,6 +1,10 @@
 {
     "dependencies": {
-        "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
+      "Microsoft.Framework.NotNullAttribute.Sources": {
+        "type": "build",
+        "version": "1.0.0-*"
+      },
+      "Newtonsoft.Json": "6.0.6"
```

Comment:
Per @Tratcher's comment, M.F.Caching.Extensions would be a better place for this